ia-guide/ia-cyoa.tw

```diff
@@ -98,13 +130,20 @@ When you are thinking about your readers, make sure you are thinking about every
 
 Now, let's [[move on->Who are you writing for?]]
 
+{back link, label: 'Or go back a step'}
+
 
 :: Writing about a car
+config.footer.left: "▰▰▱▱▱▱▱▱ 25%"
+--
+
 ## Writing about a car
 You lucky thing!
 There is a pretty good chance that you needed to learn how to do this thing yourself, and hopefully it wasn't all that long ago, and you can fairly easily cast your mind back to those days when you didn't know what this thing was, or how it worked.
 And if you didn't have to do this yourself, hopefully there is someone nearby who has, so go ask them questions!
 
+{embed image: 'images/car.svg', alt: 'A cheering person in a green outfit driving a red and white striped sports car, with the caption "big red car"'}
```

Comment:
I really love this image. I'm not across copyright law, but I'm wondering if we're borderline with the Wiggles here?Perhaps we can change the words or remove the words from the image.  "The Big Red Car" is clearly affiliated with The Wiggles, and the song "Toot toot chugga chugga big red car" is one of their biggest hits. 
The image on this page https://wiggles.fandom.com/wiki/Toot_Toot,_Chugga_Chugga,_Big_Red_Car looks very close to the hand drawn image we're using - is it okay? Are we infringing? I'm not sure.

Reply:
I wondered this too. I think there are some small edits we're going to need to make to a few of the images before we send this live.